typings-checker/global-id.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { globalId } from "../dist/src/index";
+
+const { getRootId, getLocalId } = globalId;
+
+describe('Global ID', () => {
+    describe('getRootId', () => {
+        it('throws an error when passed things other than objects', () => {
+// $ExpectError Argument of type '"wrong"' is not assignable
+            getRootId('wrong');
+
+// $ExpectError Argument of type 'null' is not assignable
+            getRootId(null);
+        });
+
+        it('throws an error when passed a props object without id', () => {
```

Comment:
getting no output for expected types when passing objects without id to getRootId.
do you know why? @amir-arad

Reply:
you mean `// $ExpectType string` fails?